bin/hsw-cli

```diff
@@ -586,99 +586,87 @@ class CLI {
+        this.log('  $ blocks: List wallet blocks.');
+        this.log('  $ change [account-name]: Derive new change address.');
+        this.log('  $ coins: View wallet coins.');
+        this.log('  $ dump [address]: Get wallet key WIF by address.');
+        this.log('  $ get: View wallet.');
         this.log('  $ history: View TX history.');
+        this.log('  $ import [wif|hex]: Import private or public key.');
+        this.log('  $ key [address]: Get wallet key by address.');
+        this.log('  $ listen: Listen for events.');
+        this.log('  $ lock: Lock wallet.');
+        this.log('  $ mktx [address] [value]: Create transaction.');
         this.log('  $ pending: View pending TXs.');
-        this.log('  $ coins: View wallet coins.');
-        this.log('  $ account list: List account names.');
-        this.log('  $ account create [account-name]: Create account.');
-        this.log('  $ account get [account-name]: Get account details.');
-        this.log('  $ address: Derive new address.');
-        this.log('  $ change: Derive new change address.');
         this.log('  $ retoken: Create new api key.');
         this.log('  $ send [address] [value]: Send transaction.');
-        this.log('  $ mktx [address] [value]: Create transaction.');
+        this.log('  $ shared add [xpubkey]: Add key to wallet.');
+        this.log('  $ resendwallet [id]: Resend pending transactions for a single wallet.');
+        this.log('  $ shared remove [xpubkey]: Remove key from wallet.');
         this.log('  $ sign [tx-hex]: Sign transaction.');
-        this.log('  $ zap [age?]: Zap pending wallet TXs.');
-        this.log('  $ abandon [hash]: Abandon a transaction.');
         this.log('  $ tx [hash]: View transaction details.');
-        this.log('  $ blocks: List wallet blocks.');
-        this.log('  $ block [height]: View wallet block.');
+        this.log('  $ unlock [passphrase] [timeout?]: Unlock wallet.');
         this.log('  $ view [tx-hex]: Parse and view transaction.');
-        this.log('  $ import [wif|hex]: Import private or public key.');
         this.log('  $ watch [address]: Import an address.');
-        this.log('  $ key [address]: Get wallet key by address.');
-        this.log('  $ dump [address]: Get wallet key WIF by address.');
-        this.log('  $ lock: Lock wallet.');
-        this.log('  $ unlock [passphrase] [timeout?]: Unlock wallet.');
-        this.log('  $ resend: Resend pending transactions.');
+        this.log('  $ zap [age]: Zap pending wallet TXs.');
+        this.log('');
+        this.log('If node is run with wallet-auth flag, wallet commands require authorization token.');
+        this.log('Admin commands require admin permissions for provided authorization token:');
+        this.log('  $ backup [path]: Backup the wallet db.');
+        this.log('  $ master: View wallet master key.');
+        this.log('  $ mkwallet [id]: Create wallet.');
```

Comment:
`mkwalle` actually does not need admin token.

Reply:
Yes, you're right. I treat it as a bug as malicious user can create a lot of wallets. Created PR: https://github.com/handshake-org/hsd/pull/581